Gemfile

```diff
@@ -41,9 +41,8 @@ end
 group :test do
   gem "govuk_schemas"
   gem "i18n-coverage"
-  gem "mocha"
```

Comment:
Did you mean to remove "mocha" in this commit as well?

Reply:
I did (we don't need it any more)

Reply:
I see that Mocha tests etc are removed later in the PR, but at this point, they're are still there. Ideally the gem would be removed after, or at the same time that the use of Mocha is removed.